content/en/security/cloud_security_management/setup/agentless_scanning/_index.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Setting up CSM Agentless Scanning
+kind: documentation
+---
+
+<div class="alert alert-info">CSM Agentless scanning is in beta.</div>
```

Comment:
"Agentless Scanning for Cloud Security Management is in public beta for AWS cloud environments."

Want to emphasize that it is for AWS only.